amlb/results.py

```diff
@@ -444,6 +444,10 @@ def compute_score(self, result=None, meta_result=None):
         required_meta_res = ['training_duration', 'predict_duration', 'models_count']
         for m in required_meta_res:
             entry[m] = meta_result[m] if m in meta_result else nan
+
+        if inference_times := Namespace.get(meta_result, "inference_times"):
+            for n_samples, measured_times in Namespace.dict(inference_times).items():
+                entry[f"inference_{n_samples}_rows"] = np.mean(measured_times)
```

Comment:
small nit: `f"inference_rows_{n_samples}"` will be easier to parse downstream via the shared prefix `inference_rows_`.

An alternative name: `infer_batch_size_{n_samples}`